pkg/scheduler/scheduler.go

```diff
@@ -279,7 +279,7 @@ func New(client clientset.Interface,
 		bindTimeoutSeconds:       options.bindTimeoutSeconds,
 		podInitialBackoffSeconds: options.podInitialBackoffSeconds,
 		podMaxBackoffSeconds:     options.podMaxBackoffSeconds,
-		enableNonPreempting:      utilfeature.DefaultFeatureGate.Enabled(kubefeatures.NonPreemptingPriority),
+		enableNonPreempting:      utilfeature.DefaultFeatureGate.Enabled(schedutil.NonPreemptingPriority),
```

Comment:
instead of importing `utilfeature "k8s.io/apiserver/pkg/util/feature"` which contains this:
https://github.com/kubernetes/apiserver/blob/master/pkg/util/feature/feature_gate.go

individual components should just define their local "DefaultFeatureGate"

Reply:
This PR is only for /pkg/features, so I will make make another PR to take care of /pkg/util/feature.